customizations/README.md

```diff
@@ -55,9 +56,13 @@ Optional configurations:
 - [Configure scale of the components](#components-scaling)
 - [Use your own bucket to store the data](#custom-buckets) (by default, GCP CARTO buckets are used)
 
-## Custom Google Cloud platform Service Account
+## Custom Service Account
 
-If you prefers using your own GCP Service Account Key, instead of the one that CARTO provides for each customer we do support it but you must  send the service account ID to CARTO support team, prior to the selfhost installation in order to grant access to the SaaS counterpart.
+For each CARTO self hosted installation, we create and provide a Google Cloud Platform Service Account key that the CARTO backend uses to authenticate requests to Google APIs.
```

Comment:
let's add -> `CARTO deploys a dedicated infrastructure for every self hosted installation and such Service Account key is required to use some of the services deployed`.